pandapipes/plotting/generic_geodata.py

```diff
@@ -3,7 +3,11 @@
 # Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
 
 import numpy as np
-from pandapipes.component_models import ExtGrid, Pipe, Sink, Source, Junction
+from pandapipes.component_models.ext_grid_component import ExtGrid
+from pandapipes.component_models.pipe_component import Pipe
+from pandapipes.component_models.sink_component import Sink
+from pandapipes.component_models.source_component import Source
+from pandapipes.component_models.junction_component import Junction
```

Comment:
The component_model \__init\__ still contains a lot of component imports. You probably mean why I didn't import it there. I am not exactly sure if I got import problems, but anyways I am of the opinion that we should be as clear as possible in our imports because using very global imports within the same package will in the end always lead to circular imports.